docs/get-started/developers/index.md

```diff
@@ -11,8 +11,71 @@ subnav:
   collection: sortedDevelopers
   order: 00
 order: 20
+importElements:
+  - rh-card
+  - rh-cta
 ---
 
+<style>
+  .grid > rh-card {
+    display: grid;
+  }
+
+  rh-card > svg[slot] {
+    fill: var(--rh-color-accent-brand-on-light, #ee0000);
+    width: var(--rh-size-icon-07, 80px);
+    margin-block-end: var(--rh-space-lg, 16px);
+  }
+
+  rh-card > p[slot],
+  rh-card > h3[slot] {
+    margin-block: 0;
+  }
+
+  #section-encapsulation,
+  #section-apis,
+  #section-framework {
+    display: grid;
```

Comment:
Instead of defining this grid here, would it be possible to use or expand upon the [grid classes](https://github.com/RedHat-UX/red-hat-design-system/blob/main/docs/styles/grid.css) Steve uses throughout the uxdot site?

Reply:
Initially, I was hesitant to expand on the grid, but you know... why not?!

Reply:
@zeroedin would love your input on the "expansion" of `.grid` to handle content + image layouts.

Reply:
As discussed on a video chat but adding my response here, these styles would be best moved to a `layouts` layer and "de-BEMified".  

The intent with the grid stylesheet was to act as a shareable artifact ie (reset, fonts, typography, base, grid) and most likely it will be replaced by some other shared style or component such as `rhx-grid` or whatever.  

Ultimately these layout styles could also inform a layout pattern that becomes a shared artifact and less ux.redhat.com specific,  IMHO they would be best served decoupled from each other.

Reply:
Thanks, @zeroedin. We'll tackle this in another PR that refactors the `layouts` and `grid` we've got in ux dot with `rhx-grid`